styles.css

```diff
@@ -584,38 +614,349 @@ textarea ::placeholder {
     display: flex;
     flex-direction: column;
     border: none;
+    background: transparent;
+    justify-content: center;
   }
 
-  .headline {
-    width: 100%;
-    height: 100%;
+  .main-background {
+    background: url('./desktopMedia/Header-llustration-desktop.svg');
     background-size: cover;
     background-repeat: no-repeat;
+    height: 100vh;
+    width: 100vw;
+    display: flex;
+    justify-content: center;
   }
```

Comment:
- [ ]  Kindly reduce the spacing (`bottom margin`) of this element so as to have a design close to the template. 👍 More so, all background-image are not displaing properly for a larger screen. 👍 

<img width="944" alt="Untitled" src="https://github.com/tan12082001/Project-portfolio/assets/103386183/a654fb04-1d43-4982-a348-6c06187d2845">